setTimeout(() => {
    console.log("Please don't waste time");
}, 1000);

try {
    console.log(chandan);
} catch (tillu) {
   console.log("you are so gorgeous"); 
}

setTimeout(() => {
    console.log("Start to clip image"); 
}, 3000);
try {
    setTimeout(() => {
        console.log(error_name);
    }, 1500);
} catch (prince) {
    console.log("Error Mil gaya");
}
setTimeout(() => {
    console.log("Make Presentation");
}, 4000);
setTimeout(() => {
    console.log("Do coding");
}, 0000);
// try catch block can't handle the asynchronous code.